test/src/test/java/org/corfudb/AbstractCorfuTest.java

```diff
@@ -102,6 +104,12 @@ public void cleanupScheduledThreads() {
         scheduledThreads.clear();
     }
 
+    @After
+    public void cleanPerTestTempDir() {
+        deleteFolder(new File(PARAMETERS.TEST_TEMP_DIR),
+                false);
+    }
+
```

Comment:
1 & 2. What is the issue with using PARAMETERS? There's a good reason for making the temporary directory a parameter, in the future, a user may choose to specify the directory through a property (for debugging, for example). I don't see a good reason it should be a function call. (maybe PARAMETERS is too long and we should shorten to PARAMS?)

3. Calling it getUniqueTempDir doesn't help. I think it'd just confuse the user, because they might think they're getting the same UniqueTempDir every time. getNewTempDir() explicitly tells the user they're getting a new temp directory whether they like it or not.

But this discussion has me conclude that we should just drop all the temporary directory functions and force the user to use the directory provided in parameters, which is what I'm doing next.